WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -513,6 +485,60 @@ - (BlogDetailsSection *)configurationSectionViewModel
     return [[BlogDetailsSection alloc] initWithTitle:title andRows:rows];
 }
 
+- (BlogDetailsSection *)externalSectionViewModel
+{
+    __weak __typeof(self) weakSelf = self;
+    NSMutableArray *rows = [NSMutableArray array];
+    BlogDetailsRow *viewSiteRow = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"View Site", @"Action title. Opens the user's site in an in-app browser")
+                                                                  image:[Gridicon iconOfType:GridiconTypeHouse]
+                                                               callback:^{
+                                                                   [weakSelf showViewSite];
+                                                               }];
+    viewSiteRow.showsSelectionState = NO;
+    [rows addObject:viewSiteRow];
+
+    if ([self shouldDisplayLinkToWPAdmin]) {
+        BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:[self adminRowTitle]
+                                                              image:[Gridicon iconOfType:GridiconTypeMySites]
+                                                           callback:^{
+                                                               [weakSelf showViewAdmin];
+                                                               [weakSelf.tableView deselectSelectedRowWithAnimation:YES];
+                                                           }];
+        UIImage *image = [[Gridicon iconOfType:GridiconTypeExternal withSize:CGSizeMake(BLogDetailGridiconAccessorySize, BLogDetailGridiconAccessorySize)] imageFlippedForRightToLeftLayoutDirection];
+        UIImageView *accessoryView = [[UIImageView alloc] initWithImage:image];
+        accessoryView.tintColor = [WPStyleGuide cellGridiconAccessoryColor]; // Match disclosure icon color.
+        row.accessoryView = accessoryView;
+        row.showsSelectionState = NO;
+        [rows addObject:row];
+    }
+
+    NSString *title = NSLocalizedString(@"External", @"Section title for the external table section in the blog details screen");
+    return [[BlogDetailsSection alloc] initWithTitle:title andRows:rows];
+}
+
+- (NSString *)adminRowTitle
+{
+    if (self.blog.isHostedAtWPcom) {
+        return NSLocalizedString(@"Dashboard", @"Action title. Noun. Opens the user's WordPress.com dashboard in an external browser.");
+    } else {
+        return NSLocalizedString(@"WP Admin", @"Action title. Noun. Opens the user's WordPress Admin in an external browser.");
+    }
+}
+
+// Non .com users and .com user whose accounts were created
+// before LastWPAdminAccessDate should have access to WPAdmin
+- (BOOL)shouldDisplayLinkToWPAdmin
+{
+    if (!self.blog.isHostedAtWPcom) {
+        return YES;
+    } else {
```

Comment:
I might be spoiled by Swift's guard now, but if the `if` is always returning we can save the `else` and avoid the extra indentation